packages/http/httpcall_common.js

```diff
@@ -5,9 +5,14 @@ makeErrorByStatus = function(statusCode, content) {
     return str.length > length ? str.slice(0, length) + '...' : str;
   };
 
+  // this solution is not browser compatible
+  // var contentToCheck = typeof Buffer != "undefined" && content instanceof Buffer ? content.toString() : content;
+  var contentToCheck = typeof content == "string" ? content : content.toString();
```

Comment:
I think it would be better to just not print the content to the console in this case? If you're using encoding: null, it's probably binary content and we probably don't want to dump an image to the console?